test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -297,6 +297,41 @@ test.describe( 'Cover', () => {
 
 		expect( isClickable ).toBe( false );
 	} );
+
+	test( 'can use focal point picker to set the focal point of the cover image', async ( {
+		editor,
+		coverBlockUtils,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+		const coverBlock = editor.canvas.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+
+		await coverBlockUtils.upload(
+			coverBlock.getByTestId( 'form-file-upload-input' )
+		);
+
+		await editor.page.keyboard.press( 'ArrowUp' );
```

Comment:
```suggestion
		await editor.selectBlocks( coverBlock );
```

Use the public api to select a block.